lein-template/src/leiningen/new/duct.clj

```diff
@@ -107,8 +110,22 @@
        (sort-by key)
        (map (fn [[path temp]] [path (render-resource data temp)]))))
 
-(defn generate-project [{:keys [vars templates dirs] :as profile}]
-  (let [data  (merge vars (select-keys profile [:deps :dev-deps]))
+(defn- parse-map [map]
+  (mapcat
+    (fn [[k v]]
+      (cond
+        (empty? v) [(str k " " v)]
+        (string? v) [(str k v)]
+        :else [(str k)
+               (clojure.string/trim-newline (with-out-str (pprint v)))]))
```

Comment:
Require `clojure.string`. Don't use it bare and rely on a quirk of the compiler.